migrations/1681244251000_stats.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export function up(pgm: MigrationBuilder): void {
+  pgm.createMaterializedView(
+    'stats_inscriptions_per_block',
+    {
+      data: true,
+    },
+    `SELECT block_height,
+      COUNT(*) AS count,
+      SUM(COUNT(*)) OVER (ORDER BY block_height) AS scan_count
```

Comment:
I'm not clear on what `scan_count` means in this context, could you explain a bit?

Reply:
I chose `scan` as the cumulative `reduce` algorithm. But it definitely sounds confusing! Going with `inscriptions` and `inscriptions_total` (for cumulative) in the new PR